scripts/binaryen-lit.in

```diff
@@ -27,7 +27,7 @@ builtin_parameters = {
     }
 }
 
-print(builtin_parameters)
+#print(builtin_parameters)
```

Comment:
intentional?

Reply:
This was extra debugging information .. I should probably just completely remove it.